### PR TITLE
Emit real time execution results

### PR DESCRIPTION
**Pull request by avsek477**
### Description:
- Add stopped variable to not process after the execution is stopped.
- Send real time execution results after each price update.

### Breaking changes:
- [ ]

### New features:
- [ ]

### Fixes:
- [ ]

### PR status:
- [ ] Version bumped
- [ ] Change-log updated
- [ ] Tests added or updated
- [ ] Documentation updated

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -37,7 +37,7 @@
   "dependencies": {
     "bfx-api-node-core": "^1.5.8",
     "bfx-api-node-util": "^1.0.2",
-    "bfx-hf-strategy": "git+https://github.com/bitfinexcom/bfx-hf-strategy.git#v1.2.3",
+    "bfx-hf-strategy": "git+https://github.com/avsek477/bfx-hf-strategy.git#update-position-details",
```
Comment (avsek477) [inline package.json:40]: update after merge(https://github.com/bitfinexcom/bfx-hf-strategy/pull/23)

File: index.js [hunk 2]
```diff
@@ -150,6 +157,11 @@ class LiveStrategyExecution extends EventEmitter {
    * @private
    */
   _enqueueMessage (type, data) {
+    debug('typppeeee %s, %s', type, this.stopped)
```
Comment (tarcisiozf) [inline index.js:160]: typo or remove debug entry

File: index.js [hunk 3]
```diff
@@ -176,6 +188,22 @@ class LiveStrategyExecution extends EventEmitter {
     this.processing = false
   }
 
+  _emitStrategyExecutionResults (type = 'candle', data) {
+    const { symbol } = this.strategyOpts
+    const { candlePrice } = this.strategyState
+    const price = type === 'candle' ? data[candlePrice] : data.price
+
+    let hasOpenPosition = false
```
Comment (tarcisiozf) [inline index.js:196]: this extra variable is not required as openPosition is already nullable and the if checks that

File: index.js [hunk 4]
```diff
@@ -266,19 +298,32 @@ class LiveStrategyExecution extends EventEmitter {
...
+    if (_isPlainObject(openPosition)) {
+      const openOrderIndex = strategyTrades.findIndex(st => st.position_id === openPosition.id)
+      if (openOrderIndex !== -1) {
+        strategyTrades[openOrderIndex] = {
+          ...strategyTrades[openOrderIndex],
+          pl: openPosition.pl
+        }
+      }
+    }
+
```
Comment (tarcisiozf) [inline index.js:326]:
```suggestion
    if (_isPlainObject(openPosition)) {
      const openOrder = strategyTrades.find(st => st.position_id === openPosition.id)
      if (openOrder) {
        openOrder.pl = openPosition.pl
      }
    }

```

**Review by tarcisiozf — APPROVED** [review]

**Review by tarcisiozf — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (vigan-abd) [reply package.json:40]:
```suggestion
    "bfx-hf-strategy": "git+https://github.com/bitfinexcom/bfx-hf-strategy.git#v1.2.4",
```

**Review by vigan-abd — APPROVED** [review]

**Pull request merged**